vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -83,13 +83,18 @@ echo "Granting access to $disk_resource_id for 1 hour"
 # shellcheck disable=SC2102
 sas=$(az disk grant-access --ids $disk_resource_id --duration-in-seconds 3600 --query [accessSas] -o tsv)
 
-# TBD: Need to investigate why `azcopy-preview login --login-type=MSI` does not work
+echo "Uploading $disk_resource_id to ${CLASSIC_BLOB}/${CAPTURED_SIG_VERSION}.vhd"
+
 echo "Setting azcopy environment variables with pool identity: $AZURE_MSI_RESOURCE_STRING"
+# TBD: Need to investigate why `azcopy-preview login --login-type=MSI` does not work for Windows
```

Comment:
@AbelHu I think this is most likely due to the fact that you're using `azcopy-preview`, not completely sure though